docs/en/observability/slo-overview.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[slo]]
+= Service-Level Objectives (SLOs) 
+
+++++
+<titleabbrev>SLOs (beta)</titleabbrev>
```

Comment:
Is it a safe assumption that users will see SLOs in the TOC and know what that means?

Reply:
This is a good question. I made it this way for consistency with the UI, but it might make more sense to spell it out the first time in the nav.